releases/nightly-build/compose-files/docker-compose-nexus-base.yml

```diff
@@ -0,0 +1,218 @@
+    networks:
+      - edgex-network
+    env_file:
+      - common.env
+    environment:
+      SERVICE_HOST: edgex-support-notifications
+    depends_on:
+      - consul
+      - database
+
+  metadata:
+    image: ${CORE_EDGEX_REPOSITORY}/docker-core-metadata-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
+    ports:
+      - "127.0.0.1:48081:48081"
+    container_name: edgex-core-metadata
+    hostname: edgex-core-metadata
+    networks:
+      - edgex-network
+    env_file:
+      - common.env
+    environment:
+      SERVICE_HOST: edgex-core-metadata
+      NOTIFICATIONS_SENDER: edgex-core-metadata
+    depends_on:
+      - consul
+      - database
+      - notifications
+
+  data:
+    image: ${CORE_EDGEX_REPOSITORY}/docker-core-data-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
+    ports:
+      - "127.0.0.1:48080:48080"
+      - "127.0.0.1:5563:5563"
+    container_name: edgex-core-data
+    hostname: edgex-core-data
+    networks:
+      - edgex-network
+    env_file:
+      - common.env
+    environment:
+      SERVICE_HOST: edgex-core-data
+    depends_on:
+      - consul
+      - database
+      - metadata
+
+  command:
+    image: ${CORE_EDGEX_REPOSITORY}/docker-core-command-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
+    ports:
+      - "127.0.0.1:48082:48082"
+    container_name: edgex-core-command
+    hostname: edgex-core-command
+    networks:
+      - edgex-network
+    env_file:
+      - common.env
+    environment:
+      SERVICE_HOST: edgex-core-command
+    depends_on:
+      - consul
+      - database
+      - metadata
+
+  scheduler:
+    image: ${CORE_EDGEX_REPOSITORY}/docker-support-scheduler-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
+    ports:
+      - "127.0.0.1:48085:48085"
+    container_name: edgex-support-scheduler
+    hostname: edgex-support-scheduler
+    networks:
+      - edgex-network
+    env_file:
+      - common.env
+    environment:
+      SERVICE_HOST: edgex-support-scheduler
+      INTERVALACTIONS_SCRUBPUSHED_HOST: edgex-core-data
+      INTERVALACTIONS_SCRUBAGED_HOST: edgex-core-data
+    depends_on:
+      - consul
+      - database
+
+  app-service-rules:
+    image: ${REPOSITORY}/docker-app-service-configurable${ARCH}:${APP_SERVICE_VERSION}
+    ports:
+      - "127.0.0.1:48100:48100"
+    container_name: edgex-app-service-configurable-rules
+    hostname: edgex-app-service-configurable-rules
+    networks:
+      - edgex-network
+    env_file:
+      - common.env
+    environment:
+      EDGEX_PROFILE: rules-engine
+      SERVICE_HOST: edgex-app-service-configurable-rules
+      SERVICE_PORT: 48100
+      MESSAGEBUS_SUBSCRIBEHOST_HOST: edgex-core-data
+      BINDING_PUBLISHTOPIC: events
+    depends_on:
+      - consul
+      - data
+
+  rulesengine:
+    image: emqx/kuiper:${KUIPER_VERSION}
+    ports:
+      - "127.0.0.1:48075:48075"
+      - "127.0.0.1:20498:20498"
+    container_name: edgex-kuiper
+    hostname: edgex-kuiper
+    networks:
+      - edgex-network
+    environment:
+      # KUIPER_DEBUG: "true"
```

Comment:
commented out means the default value is `true`? or the default value is false?

Reply:
Default is `false`. If you want DEBUG logs then you un-comment this line.